src/libraries/shims/Directory.Build.props

```diff
@@ -4,59 +4,39 @@
 
   <PropertyGroup>
     <TargetFramework>$(NetCoreAppCurrent)-$(TargetOS)</TargetFramework>
-    <Nullable>disable</Nullable>
+    <EnableDefaultItems>true</EnableDefaultItems>
     <!-- By default make shims compile against reference assemblies. -->
     <CompileUsingReferenceAssemblies>true</CompileUsingReferenceAssemblies>
-    <IsPartialFacadeAssembly>true</IsPartialFacadeAssembly>
-    <GenFacadesIgnoreMissingTypes>true</GenFacadesIgnoreMissingTypes>
-    <!-- ensure the desktop compat shims reference the lowest possible version of dependencies
-         since those do not all ship as part of the framework and we don't want to force apps
-         to reference the latest packages.
-         netstandard.dll doesn't need to do this since it has no dangling dependencies -->
-    <GenFacadesForceZeroVersionSeeds Condition="'$(MSBuildProjectName)' != 'netstandard'">true</GenFacadesForceZeroVersionSeeds>
+    <!-- Ensure the .NET Framework shims reference the lowest possible version of dependencies since
+         those do not all ship as part of the framework and we don't want to force apps to reference the
+         latest packages. netstandard.dll doesn't need to do this since it has no dangling dependencies. -->
```

Comment:
Why do we not want the latest package to he referenced?

Reply:
This is already the same in main. Or are you asking about why we are doing this generally? IMO the comment captures that nicely. We make sure that shims don't depend on the very latest packages and instead work with any version. When you look at any of the shims in ilspy, i.e. mscorlib you will see that all dependency versions in its metadata are 0.0.0.0.